keyvalues/keyvalues.go

```diff
@@ -39,3 +40,14 @@ func Parse(src []string, allowEmptyValues bool) (map[string]string, error) {
 	}
 	return results, nil
 }
+
+// Join returns a sorted slice containing each item in the given map, as
+// "key=value" strings.
+func Join(m map[string]string) []string {
+	var result []string
+	for k, v := range m {
+		result = append(result, fmt.Sprintf("%s=%s", k, v))
```

Comment:
Do we want to enforce uniqueness?

Reply:
Not sure what you mean.. map keys are unique.